metrics/nist/app.py

```diff
@@ -0,0 +1,6 @@
+import evaluate
+from evaluate.utils import launch_gradio_widget
+
+
+module = evaluate.load("nist")
```

Comment:
I wonder if we should change the name to something like `load("nist_mt")`? What if there is a second metric from NIST we want to add in the future?

Reply:
Good idea! I only have experience with NIST for MT but if they have other metrics it makes sense indeed.